tests/conftest.py

```diff
@@ -62,6 +63,11 @@ def test_aws_user():
         yield client.create_access_key(UserName="test-user")["AccessKey"]
 
 
+@pytest.fixture
+def article_class():
+    return Article(doi="111.1/1111")
```

Comment:
I might propose renaming this fixture.  What you get back is an Article instance.  What about something boring but descriptive like `sample_article` or `sample_article_111_1111`?

Reply:
Correcting this mistaken usage across several fixtures